extensions/observability-devservices/sub/lgtm/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
rename it to `observability-devservices/devservices/lgtm/`? instead of `observability-devservices/sub/lgtm/`

Reply:
Well, there will be many of such similar modules / dirs, with only pom.xml inside,
and I don't want to "polute" the whole `devservices` module, hence `sub`.
So that `devservices` only mostly has the usual suspects - deployment, runtime, common, ...

Reply:
it's just because `sub` folder has no meaning and we should try to improve it either by renaming it or moving somewhere.

Reply:
OK, I see what you mean ... let's rename it then.
Any idea(s)?
This is where simplified extension poms will live ...

Reply:
sink, apm, user-side, service?

Reply:
I like sink, although dunno how is that better then sub. :-)
What is apm?
Nah on the other two ...

Reply:
Application performance monitoring (APM)... Dynatrace, Grafana and New Relic, they are all APMS

Reply:
Lets go with sink then?

Reply:
sure

Reply:
Done